### PR TITLE
Fix for 1325

### PR DESCRIPTION
**Pull request by sbrossie**

### CODE REVIEW

File: invoice/src/main/java/org/killbill/billing/invoice/usage/ContiguousIntervalConsumableUsageInArrear.java [hunk 1]
```diff
@@ -61,7 +61,7 @@
 
 import static org.killbill.billing.invoice.usage.UsageUtils.getConsumableInArrearTieredBlocks;
 
-public class ContiguousIntervalConsumableUsageInArrear extends ContiguousIntervalUsageInArrear {
+    public class ContiguousIntervalConsumableUsageInArrear extends ContiguousIntervalUsageInArrear {
```
Comment (pierre) [inline invoice/src/main/java/org/killbill/billing/invoice/usage/ContiguousIntervalConsumableUsageInArrear.java:64]: Nit: extra space.

File: invoice/src/main/java/org/killbill/billing/invoice/usage/ContiguousIntervalConsumableUsageInArrear.java [hunk 2]
```diff
@@ -242,20 +246,19 @@ UsageConsumableInArrearTierUnitAggregate computeToBeBilledConsumableInArrearWith
...
                 targetBlock = tieredBlock;
                 targetTierNum = tierNum;
                 break;
             }
         }
         final int lastBlockTierSize = targetBlock.getSize().intValue();
-        final int nbBlocks = units.intValue() / lastBlockTierSize + (units.intValue() % lastBlockTierSize == 0 ? 0 : 1);
-
-        return new UsageConsumableInArrearTierUnitAggregate(targetTierNum, targetBlock.getUnit().getName(), targetBlock.getPrice().getPrice(getCurrency()), targetBlock.getSize().intValue(), nbBlocks);
+        final long nbBlocks = units.intValue() / lastBlockTierSize + (units.intValue() % lastBlockTierSize == 0 ? 0 : 1);
```
Comment (pierre) [inline invoice/src/main/java/org/killbill/billing/invoice/usage/ContiguousIntervalConsumableUsageInArrear.java:260]: Do we want `units.intValue();` here?

Comment (sbrossie) [reply invoice/src/main/java/org/killbill/billing/invoice/usage/ContiguousIntervalConsumableUsageInArrear.java]: Nope, you right, fixed in https://github.com/killbill/killbill/pull/1328/commits/c3548aede500713383b62773b5c7a9e4d7e2f1a4

**Comment by sbrossie** [comment]
@msqr Any chance you could verify if this fix helps with your overflow?

**Comment by msqr** [comment]
I did a small set of tests and it does seem to work, however I have not done a thorough validation.

**Pull request merged**